src/Install/CodeEnvironment/OpenCode.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laravel\Boost\Install\CodeEnvironment;
+
+use Laravel\Boost\Contracts\Agent;
+use Laravel\Boost\Contracts\McpClient;
+use Laravel\Boost\Install\Enums\McpInstallationStrategy;
+use Laravel\Boost\Install\Enums\Platform;
+
+class OpenCode extends CodeEnvironment implements Agent, McpClient
+{
+    public function name(): string
+    {
+        return 'opencode';
+    }
+
+    public function displayName(): string
+    {
+        return 'OpenCode';
+    }
+
+    public function systemDetectionConfig(Platform $platform): array
+    {
+        return match ($platform) {
+            Platform::Darwin, Platform::Linux => [
+                'command' => 'which opencode',
```

Comment:
`which` can sometimes not be available, whereas `command` is a built-in
```suggestion
                'command' => 'command -v opencode',
```